wagtail_content_import/templatetags/wagtailcontentimport_tags.py

```diff
@@ -17,3 +18,11 @@ def wagtailcontentimport_pickerjs(context):
         js_snippets.append(picker.render_js_init(context['request']))
 
     return mark_safe('\n'.join(js_snippets))
+
```

Comment:
Should have two spaces between module-level functions

Reply:
Have added, thanks!